backend/src/packages/users/user.repository.ts

```diff
@@ -2,6 +2,13 @@ import { type Repository } from '#libs/types/types.js';
 import { UserEntity } from '#packages/users/user.entity.js';
 import { type UserModel } from '#packages/users/user.model.js';
 
+import { UsersRelation } from './libs/enums/users-relation.enum.js';
```

Comment:
```suggestion
import { UsersRelation } from './libs/enums/enums.js';
```

Reply:
So for repeating such mistakes. I'm just hurying(